doc/tutorials/12-constant_pH/12-constant_pH.ipynb

```diff
@@ -120,20 +196,75 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "K = 1e-4\n",
-    "pK = -np.log10(K)"
+    "# create the particles\n",
+    "##################################################\n",
+    "# we need to define bonds before creating polymers\n",
+    "from espressomd.interactions import HarmonicBond\n",
+    "hb= HarmonicBond(k=30, r_0=1.0)\n",
+    "system.bonded_inter.add(hb)\n",
+    "\n",
+    "# create the polymer composed of ionizable acid groups, initially in the ionized state\n",
+    "from espressomd import polymer\n",
+    "polymers = polymer.positions(n_polymers=1,\n",
+    "                             beads_per_chain=N_acid,\n",
+    "                             bond_length=0.9, seed=23)\n",
+    "for p in polymers:\n",
+    "    for i, m in enumerate(p):\n",
+    "       id = len(system.part)\n",
```

Comment:
I think if you can come up with more expressive variable names it would help to make it more readable:
*e.g.*: 
``` python
for p in polymer:
    for index, monomer_pos in enumerate(p):
```